app/addons/config/assets/less/config.less

```diff
@@ -112,6 +140,61 @@ table.config {
   }
 }
 
+#add-option-popover {
+  background-color: #333;
+  color: white;
+  padding: 20px;
+  border-radius: 0;
+  width: 300px;
+  max-width: none;
+  display: block;
+
+  .popover-content {
+    padding: 0;
+  }
+
+  .popover-title {
+    font-size: 16px;
+    padding: 0;
+    margin: 10px 0;
+    border: none;
+    display: block;
+    text-align: left;
+    color: #fff;
+    text-shadow: none;
+    height: auto;
+    line-height: 1em;
+    background-color: #333;
+  }
+
+  input {
+    width: 100%;
+  }
+
+  a.btn {
+    color: white;
+    background-color: @linkColor;
+    line-height: 1.5em;
+    border: 0;
+    padding: 10px 10px 9px;
+    font-size: 14px;
+    .border-radius(5px);
+
+    &:hover {
+      background-color: #cbcbcb;
+      color: white;
+    }
+  }
+}
+
+.popover.bottom .arrow, .popover.bottom .arrow::after {
+  border-bottom-color: #333 !important;
```

Comment:
1) do you want to change all popovers in fauxton?

2) by making a more specific css selector you can avoid the `!important`.